### PR TITLE
feat: sql config .tfvars

### PR DESCRIPTION
**Pull request by m-juckes-pins**
## Describe your changes

- Creating sql config for each env
- groups have been created in azure portal

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: infrastructure/database-monitoring.tf [hunk 1]
```diff
@@ -117,7 +117,7 @@ resource "azurerm_monitor_metric_alert" "sql_db_cpu_alert" {
   }
 
   action {
```
Comment (m-juckes-pins) [inline infrastructure/database-monitoring.tf:119]: Not 100% if targeting the desired id 'crown_tech'

File: infrastructure/environments/dev.tfvars [hunk 2]
```diff
@@ -34,6 +34,24 @@ front_door_config = {
   use_tooling = true
 }
 
+sql_config = {
+  admin = {
+    login_username = "pins-crown-sql-dev"
```
Comment (m-juckes-pins) [inline infrastructure/environments/dev.tfvars:39]: created groups for each env and added owners of them. Nothing more at this point.

File: infrastructure/environments/prod.tfvars [hunk 3]
```diff
@@ -34,6 +34,24 @@ front_door_config = {
   use_tooling = true
 }
 
+sql_config = {
+  admin = {
+    login_username = "pins-crown-sql-prod"
+    object_id      = "00d052c3-0a51-4f7d-93fc-7c366877aed6"
+  }
+  sku_name    = "Basic"
+  max_size_gb = 2
```
Comment (beejjacobs) [inline infrastructure/environments/prod.tfvars:43]: We'll need to adjust these, but can do a separate PR for that.

**Review by beejjacobs — APPROVED** [review]

**Review by beejjacobs — DISMISSED** [review]

**Review by beejjacobs — APPROVED** [review]
